### DISCUSSION TITLE
Can this be a dotnet tool?

### DISCUSSION THREAD

**Discussion by malgreen** [kuiperzone/PupNet-Deploy#6] [Announcements]
I think this would would be _even_ better if it was packaged as a dotnet tool (see [this](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools-how-to-create)), and uploaded to NuGet.
Short term this could make installation easier, and longer term this could lead to PupNet being easily used in GitHub Actions for Continuous Delivery - AFAIK InnoSetup is also installed on the GitHub Actions Windows runner.

**Comment by kuiperzone** [comment]
Wow! I wasn't actually aware of this. Will look into it.

**Comment by malgreen** [comment]
Sounds great, thanks!

**Comment by kuiperzone** [comment]
OK. Done it. See below and version 1.2 for new features. Thanks for the suggestion.

https://www.nuget.org/packages/KuiperZone.PupNet

**Reply by malgreen** [reply]
FYI, I have implemented a CI/CD Github Actions workflow for my application [here](https://github.com/Malgreen/NexNux/blob/main/.github/workflows/release.yml), which can hopefully be used for reference for other people.
It creates a release, builds packages with PupNet, and uploads them to the release.
